src/plone/recipe/zope2instance/recipe.py

```diff
@@ -913,22 +914,48 @@ def build_wsgi_ini(self):
             "profile_flush_at_shutdown": profile_flush_at_shutdown,
             "profile_unwind": profile_unwind,
         }
-        global wsgi_ini_template
+
+        # Check custom wsgi-ini-template and wsgi-logging-ini-template
         wsgi_ini_template_path = self.options.get("wsgi-ini-template")
+        wsgi_logging_ini_template_path = self.options.get("wsgi-logging-ini-template")
+        if wsgi_ini_template_path and wsgi_logging_ini_template_path:
+            raise ValueError("wsgi_ini_template_path and wsgi_logging_ini_template_path cannot be used together")
+
+        global wsgi_ini_template, wsgi_logging_ini_template
+
+        # Load wsgi template from file or use global var as default
         if wsgi_ini_template_path:
             try:
                 with open(wsgi_ini_template_path) as fp:
                     wsgi_ini_template = fp.read()
             except OSError:
                 raise
 
+        # Load wsgi logging template from file or use global var as default
+        elif wsgi_logging_ini_template_path:
+            try:
+                with open(wsgi_logging_ini_template_path) as fp:
+                    wsgi_logging_ini_template = fp.read()
+            except OSError:
+                raise
+
+            # Add custom wsgi logging template to wsgi template
+            wsgi_ini_template += wsgi_logging_ini_template
```

Comment:
Can you beautify this and line 946 that read exactly the same ? (As discussed in flesh.)